pkgs/applications/editors/neovim/default.nix

```diff
@@ -80,6 +80,7 @@ in
       "-DGPERF_PRG=${gperf}/bin/gperf"
     ]
     ++ optional doCheck "-DBUSTED_PRG=${neovimLuaEnv}/bin/busted"
+    ++ optional (lua.pname or "" == "lua") "-DPREFER_LUA=ON"
```

Comment:
better use the already existing `isLuaJIT `

Reply:
Thanks!  I didn't think of looking in `lua.pkgs` for this attribute.

Reply:
I might move it to lua.passthru as in python